src/CxbxKrnl/Emu.cpp

```diff
@@ -267,33 +290,59 @@ bool TryHandleException(EXCEPTION_POINTERS *e)
 	if (EmuX86_DecodeException(e)) {
 		// We're allowed to continue :
 		return true;
-	}
+	}
 
-	// Try to Report this exception to the debugger, which may allow handling of this exception
-	if (CxbxDebugger::CanReport()) {
-		bool DebuggerHandled = false;
-		CxbxDebugger::ReportAndHandleException(e->ExceptionRecord, DebuggerHandled);
-		if (DebuggerHandled) {
-			// Bypass exception
-			return false;
-		}
+	genericException(e);
+
+	// We do not need EmuException to handle it again.
+	bOverrideException = true;
 
-		// Kill the process immediately without the Cxbx notifier
-		EmuExceptionExitProcess();
-	}
-	else {
-		// notify user
-		EmuExceptionNonBreakpointUnhandledShow(e);
-	}
+	// Unhandled exception :
+	return false;
+}
+
+// Only for LLE emulation coding (to help performance a little bit better)
+LONG NTAPI lleException(EXCEPTION_POINTERS *e)
+{
+	g_bEmuException = true;
+	LONG result = lleTryHandleException(e) ? EXCEPTION_CONTINUE_EXECUTION : EXCEPTION_CONTINUE_SEARCH;
+	g_bEmuException = false;
+	return result;
+}
+
+// Only for Cxbx emulation coding (to catch all of last resort exception may occur.)
+bool EmuTryHandleException(EXCEPTION_POINTERS *e)
+{
+
+	// Check if lle exception is already called first before emu exception.
+	if (bOverrideException) {
+		return false;
+	}
+
+	// Make sure access-violations reach EmuX86_DecodeException() as soon as possible

```

Comment:
EmuX86_DecodeException doesn't occur here anymore

Reply:
Hmm... thanks for your eagle 🦅 eye 👁.
😃